index.html

```diff
@@ -278,19 +278,23 @@ <h4>Game streaming</h4>
         <tr>
            <td>N15</td>
            <td>The application must be able to take steps to ensure a low and consistent
-             latency for audio, video and data under varying network conditions. This may
-             include tweaking of transport parameters for both media and data.</td>
+           latency for audio, video and data under varying network conditions. This may
+	   include tweaking of transport parameters for both media and data.</td>
         </tr>
         <tr>
            <td>N37</td>
            <td>It must be possible for the user agent's receive pipeline to process
-           video at high resolution and framerate (e.g. without copying raw video  
-           frames).</td>  
+           video at high resolution and framerate (e.g. without copying raw video
+           frames).
+	   To be specific, 120FPS, 165 FPS, 240FPS are all supported by cloud gaming and desirable to bring to browser too.
+	   Higher Resolution support up to 4K and High Dynamic Range(HDR) are also desiable for the support.</td>
         </tr>
         <tr>
            <td>N38</td>
            <td>The application must be able to control the jitter buffer and rendering
-           delay.</td>
+           delay. To be specific to the Cloud Gaming Application, Maintaing the steady latency within a range
+	   is critical for player adaptation. Good latency ranges are between 30ms - 150ms.
+	   Beyond 170ms is detrimental to user experience. </td>
```

Comment:
I don't think supporting a certain frame rate or latency can be a requirement, because that depends on system, network and implementation.

Is it fair to say that the problem here isn't that "the implementation needs to be better!", rather that there is a tradeoff between more or less real-time use cases?

I'm wondering if the requirement should be phrased more around the app being able to influence the implementation to prioritize what is more important to the application?

Reply:
What kind of API proposals would be addressing the cloud gaming use case? Are we talking about control knobs for the jitter buffer or forcing HW encoding?

Reply:
@henbos  thanks for the feedback. Will update the PR with more detailed clarifications for the cloud gaming.